SecurityPkg/Test/Mock/Library/GoogleTest/Ppi/MockTcgPpiLib/MockTcgPpiLib.cpp

```diff
@@ -0,0 +1,15 @@
+/** @file MockTcgPpiLib.cpp
+  Google Test mocks for TcgPpi
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+#include <GoogleTest/Ppi/MockTcgPpiLib.h>

```

Comment:
We actually don't need to have a cpp or inf file for a PPI mock. Please move the mock definitions to the header file.

Reply:
Hi Vivian,
I just pushed a commit to remove them. Please help to review it.

Thanks,
Paddy